source/guide_wekan.rst

```diff
@@ -0,0 +1,196 @@
+
+::
+
+ [isabell@stardust ~]$ $ uberspace tools version use node 12
+ Selected Node.js version 12
+ The new configuration is adapted immediately. Minor updates will be applied automatically.
+ [isabell@stardust ~]$
+
+The domain you want to use should be already set up:
+
+.. include:: includes/web-domain-list.rst
+
+We'll also need :lab:`MongoDB <guide_mongodb>`, so follow the MongoDB guide and come back when it's running.
+
+Let's create a password for a new database:
+
+.. code-block::
+
+ [isabell@stardust ~]$ < /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-15};echo;
+ randompassword
+ [isabell@stardust ~]$
+
+Once we have that, let's create the new database for Wekan itself.
+
+.. code-block:: 
+
+ [isabell@stardust ~]$ mongo
+ MongoDB shell version v4.4.1
+ connecting to: mongodb://127.0.0.1:21017/admin
+ Implicit session: session { "id" : UUID("6fd371f6-e1fa-461c-be0c-ea3cbe230a01") }
+ MongoDB server version: 4.4.1
+ > use wekan
+ > db.createUser(
+    {
+      user: "wekan",
+      pwd: "randompassword",
+      roles: ["readWrite"]
+    }
+   )
+ > exit
+
+Installation
+============
+
+Download and extract .ZIP archive
+-----------------------------------
+
+Check the Wekan website for the `latest release`_ and copy the download link to the wekan-*.*.zip file. Then use ``wget`` to download it. Replace the URL with the one you just copied.
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ wget https://releases.wekan.team/wekan-4.43.zip
+ […]
+ Saving to: ‘wekan-4.43.zip’
+
+ 100%[========================================================================================================================>] 3,172,029   3.45MB/s   in 0.9s
+
+ 2018-10-11 14:48:20 (3.45 MB/s) - 'wekan-4.43.zip' saved [3172029]
+ [isabell@stardust ~]$
+
+Unzip the archive and then delete it. Replace the version in the file name with the one you downloaded.
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ unzip wekan-4.43.zip
+ [isabell@stardust ~]$ rm wekan-4.43.zip
+ [isabell@stardust ~]$
+
+
+Install server
+--------------
+
+Change into the server folder of wekan and delete two files before installing all dependencies and compile the code using node.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd bundle/programs/server
+ [isabell@stardust ~]$ rm node_modules/.bin/node-gyp
+ [isabell@stardust ~]$ rm node_modules/.bin/node-pre-gyp 
+ [isabell@stardust server]$ npm install node-gyp node-pre-gyp fibers
+ [isabell@stardust server]$ cd
+ [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+Configure Webserver
+-------------------
+
+Now, the server is ready to be started. For that, we need to start the server with environmental variables of the database we created earlier
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ PORT=8080 MONGO_URL="mongodb://wekan:randompassword@127.0.0.1:27017/wekan" ROOT_URL="https://isabell.uber.space" node bundle/main.js
```

Comment:
Please remove the manual start because there is no interactive input necessary. Users will notice during the "Now let’s start the service" step, if something doesn't work.